### PR TITLE
Update device-limit-intune-azure.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to remove the conflicting information.

Reference -https://docs.microsoft.com/en-us/mem/intune/enrollment/device-enrollment-program-enroll-ios

user affinity is supported in  iOS/macOS ADE

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/memdocs/issues/2031

**Comment by PRMerger4** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and merge this PR. Thanks

**Comment by yogkumgit** [comment]
@lenewsad Gentle reminder, Please merge this PR. Thanks

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
Thanks @msbemba! Made some minor changes to fix table formatting. 

#sign-off

File: memdocs/intune/enrollment/device-limit-intune-azure.md [hunk 1]
```diff
@@ -58,25 +58,24 @@ Azure device limit restrictions set the maximum number of devices that either Az
...
 
-| Device platform | User affinity | Azure applies | Intune applies |
-| ----- | ----- | ----- | ----- | ----- |
-| Android Enterprise personally-owned work profile | Yes | Yes | Yes|
-| Android Enterprise dedicated device | No | No | No |
-| Android Enterprise fully managed | Yes | Yes | Yes |
-| Android Enterprise corporate-owned work profile | Yes | Yes | Yes |
-| Android device administrator | Yes | Yes | Yes |
+| Device platform | User affinity | Azure applies | Intune applies |  
+| ----- | ----- | ----- | ----- | ----- |
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/device-limit-intune-azure.md:62]:
```suggestion
| ----- | ----- | ----- | ----- |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, the table does not render correctly. Committing this suggested change should fix it. Thanks.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by PRMerger-2** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**